src/oidc.ts

```diff
@@ -0,0 +1,106 @@
+import { writeFile } from 'fs/promises';
+import { Issuer, TokenSet } from 'openid-client';
```

Comment:
Doesn't our linter usually require us to break multiple imports into separate lines? It's OK either way, though.

Reply:
Only when the line gets too long! I believe it spills at 100 characters.